charts/umbrella/values.yaml

```diff
@@ -1321,3 +1324,50 @@ bdrs-server-memory:
         className: "nginx"
         tls:
           enabled: false
+
+ssi-dim-wallet-stub:
+  enabled: false
+  wallet:
+    replicaCount: 1
+    host: ssi-dim-wallet-stub.tx.test
+    nameSpace: "umbrella"
+    appName: "ssi-dim-wallet-stub"
+    configName: "ssi-dim-wallet-config"
+    serviceName: "ssi-dim-wallet-service"
+    secretName: "ssi-dim-wallet-secret"
+    ingressName: "ssi-dim-wallet-ingress"
+    image:
+      repository: tractusx/ssi-dim-wallet-stub
+      pullPolicy: IfNotPresent
+      tag: "latest"
```

Comment:
do you use latest because this change https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/commit/58a29fae4172691a0efa6a4f25bbf9e9418240cc isn't part of release yet?

Reply:
At the moment only the 'latest' and 'main' versions are [available](https://hub.docker.com/r/tractusx/ssi-dim-wallet-stub/tags).

Reply:
understood.
Hi @nitin-vavdiya could you please a 0.1.2 release of the stub?

Reply:
Sure, 
I need to check the publish docker image workflow how we can add tags.
@CDiezRodriguez  Can you help here to update workflow?

Reply:
Hi @nitin-vavdiya after the release of the helm chart, you need to push a tag with [this pattern,](https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/blob/main/.github/workflows/docker-build-and-publish.yml#L28) so in this case - I assume - 0.0.2 because [your current app Version is 0.0.1](https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/blob/main/charts/ssi-dim-wallet-stub/Chart.yaml#L26) and you'd raise it as part of the release, then the workflow should take care of the rest https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/blob/main/.github/workflows/docker-build-and-publish.yml#L71

Reply:
Yes, I would be happy to

Reply:
Still, we don't have a docker image with tag 0.1.2

Reply:
Do you mean create tag in repo?
Yes, I just created it and we have docker image with tag 0.0.2: https://hub.docker.com/layers/tractusx/ssi-dim-wallet-stub/0.0.2/images/sha256-6c1a6b658dd34559ba063d7a32d845dd517f1cccaacd7f799d5e6ae418843f08?context=explore

Reply:
I used the new image, and it works fine locally for me.